### DISCUSSION TITLE
Is column level ACL possible? [Feb 16]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1335] [Discord Threads]
> 🙋‍♂️ **monkeyz**

**Comment by dstala** [comment]
> 💁 **o1lab**  
#### Not yet  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/943164089851605114)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)

**Comment by gemurdock** [comment]
Would be an important feature
